testdata/script/stack_submit_default_update_only.txt

```diff
@@ -0,0 +1,42 @@
+# stack submit with spice.stackSubmit.updateOnly so only updated branches are submitted.
+
+as 'Test <test@example.com>'
+at '2025-07-25T09:18:32Z'
+
+# setup
+cd repo
+git init
+git commit --allow-empty -m 'Initial commit'
+git config spice.stackSubmit.updateOnly true
+
+# set up a fake GitHub remote
+shamhub init
+shamhub new origin alice/example.git
+shamhub register alice
+git push origin main
+
+env SHAMHUB_USERNAME=alice
+gs auth login
+
+# create a stack with the first branch submitted
+git add feature.txt
+gs bc feature -m 'Add feature'
+gs bs
+git add feature2.txt
+gs bc feature2 -m 'Add feature 2'
+gs down
+git add feature-update.txt
+git commit --amend
+gs sr
+
+# run stack submit and ensure it only updates the first branch
+gs ss
+
```

Comment:
I'm not sure how to verify that we didn't upload the second branch here.

Reply:
I think you could copy this test pretty much verbatim: [`submit_update_only.txt`](https://github.com/abhinav/git-spice/blob/main/testdata/script/submit_update_only.txt). I think the way to test this is by running `gs ll` after `gs ss`, which will show that the second branch does not have a CR number next to the branch name.

Reply:
^ and logs. I updated to use `gs ls` and to match log output.